trackma/lib/libanilist.py

```diff
@@ -482,7 +483,14 @@ def _parse_info(self, item):
                 ('Average score',   item.get('averageScore')),
                 ('Status',          self._translate_status(item['status'])),
             ]
-        })
+        }
+        if item.get('nextAiringEpisode', None):
+            next_episode_release_time = self._int2date(item.get('nextAiringEpisode').get('airingAt'))
+            time_diff  = next_episode_release_time - datetime.datetime.now()
+            show_info['extra'].append(('Next Episode', item.get('nextAiringEpisode').get('episode')))
+            show_info['extra'].append(('Next Episode In', str(time_diff)))
+        show_info['extra'].append(('Information fetched on', datetime.datetime.now().strftime("%d %B, %Y at %I:%M %p")))
```

Comment:
From your testing or knowledge, how ofting is this information refetched? It doesn't really help if this needs to be requeried manually. Perhaps it should even be invalidated/ignored if the timestamp is in the past already.

Reply:
Probably makes more sense to ensure that UIs just show the information in the info view, rather than hack it onto the `extra` field on a per-API basis.